components/Wrappers/AppData.tsx

```diff
@@ -1,7 +1,9 @@
-import React, { createContext, useContext, useState } from 'react';
+import React, { createContext, useContext, useEffect, useState } from 'react';
 
 import { User, Wallet, Privy } from '~/types/appData';
-import { Token } from '~/types/token';
+import { Token, UserData } from '~/types/supabaseTypes';
+
+import { useSupabaseSubscription } from '~/services/Supabase/useSupabaseSubscription';
 
 interface ConfigType {
   user: User;
```

Comment:
This is the privy user right? If so, and if its not a duplicate of our new `userData` object, it might be wise to rename it to `privyUser` or something to avoid confusion

Reply:
It is not related to Privy, but instead contains data from the `Users` table. You are correct that this is confusing and should be renamed.